submission/pipelines/generate_pipelines.py

```diff
@@ -10,9 +11,12 @@
 from byudml.metafeature_extraction.metafeature_extraction import MetafeatureExtractor
 from byudml import __imputer_version__, __imputer_path__,  __metafeature_version__,  __metafeature_path__
 import sys
-sys.path.append('./submission')
-from utils import get_new_d3m_path, clear_directory, write_pipeline_for_submission
+sys.path.append('.')
```

Comment:
do we need all of `.` or is `./submission` sufficient?

Reply:
When I ran this with './submission' it didn't run correctly so I changed it back to what I had.  I think all we need that for is to import utils.py though

Reply:
interesting...